addons/portal_rating/static/src/js/portal_composer.js

```diff
@@ -36,6 +36,7 @@ PortalComposer.include({
             'default_message_id': false,
             'default_rating_value': 4.0,
             'force_submit_url': false,
+            'send_button_label': _t("Send"),
```

Comment:
This code needs to go in the "portal" module, that way this option is always defaulted to "Send".

Currently if you only install portal but not portal_rating, the option will not exist and the button label will be "false" or "undefined".